requirements.txt

```diff
@@ -1,5 +1,6 @@
-markdown==3.3.4
```

Comment:
Having this pinned was causing a bit of trouble, but if you want it pinned, I can track down the updated version that works and pin it to that.